### ISSUE TITLE
Is execution service implementation available?

### ISSUE THREAD

**Issue by ligeng0197** [uw-mad-dash/llamatune#1]
Thank the authors for such an inspiring and practical database knob tuning work! I read the majority of logits of this code repository and find the execution service logits in grpc absent. Since we meet some metric fluctuating problems in taking very similar benchmark experiments with benchbase like this code repository setting, we really appreciate any insights about the evaluation service execution details or how to keep workload execution metric stable and repeatable. Thank Kanellis and co-authors for this valuable work!
